api/questions/create.js

```diff
@@ -0,0 +1,53 @@
+const generateRequest = require('../lib/request');
+const generatePostRequest = require('../lib/postRequest');
+const config = require('../../config');
+
+function postQuestion(hearingId, headers, body) {
+    return generatePostRequest(`${config.services.coh_cor_api}/continuous-online-hearings/${hearingId}/questions`, {headers, body})
+}
+
+function postHearing(caseId, userId, headers, jurisdictionId = 'SSCS') {
+    const body = {
+        case_id: caseId,
+        jurisdiction: jurisdictionId,
+        panel: [{identity_token: 'string', name: userId}],
+        start_date: (new Date()).toISOString()
+    };
+    
+    return generatePostRequest(`${config.services.coh_cor_api}/continuous-online-hearings`, {headers, body})
+        .then(hearing => hearing.online_hearing_id)
+}
+
+function getHearingId(caseId, userId, headers) {
+    return generateRequest(`${config.services.coh_cor_api}/continuous-online-hearings?case_id=${caseId}`, {headers})
+        .then(hearing => hearing.online_hearings[0] ? hearing.online_hearings[0].online_hearing_id : postHearing(caseId, userId, headers));
+}
+
+function formatQuestion(body, userId) {
+    return {
```

Comment:
Are we defining a schema here? Seems like this should all come from the COR API?